test/filesystemevent/main.d

```diff
@@ -48,135 +48,6 @@ import ocean.task.Task;
 
 import ocean.task.Scheduler;
 
-deprecated class FileModificationTestTask: Task
```

Comment:
Yeah, I wouldn't pay much attention on testing deprecated modules, most modules are just added to the ignore list anyway. Is not the ideal but as long as we don't have a better method to test deprecated stuff, I wouldn't bother much.